core/src/components/toggle/toggle.tsx

```diff
@@ -131,6 +132,12 @@ export class Toggle implements ComponentInterface {
     this.emitStyle();
   }
 
+  get onOffLabelsEnabled(): boolean {
+    const { enableOnOffLabels } = this;
+
+    return enableOnOffLabels !== undefined ? enableOnOffLabels : config.get('onOffLabelsEnabled');
+  }
+
```

Comment:
Can we update the `enableOnOffLabels` to default to `config.get('onOffLabelsEnabled')` instead? This lets us avoid the getter and the updates in the `render` function.

Your tests would change to check the `enableOnOffLabels` property instead.

Reply:
Will this still work & be correctly re-rendered if user's update this config value while the app runs? 🤔

I orientated on the [`backButtonIcon`](https://github.com/ionic-team/ionic-framework/blob/611832b0d51da295c1bf2897972c4e8baf6e23a3/core/src/components/back-button/back-button.tsx#L80)

Reply:
The global config is not designed to be reactive, so that behavior is out of scope. Developers will be able to manually override `enabledOnOffLabels` and have that be reactive though.